FileIO/VtkIO/VtuInterface.h

```diff
@@ -38,13 +38,20 @@ class VtuInterface
 	~VtuInterface();
 
 	/// Read an unstructured grid from a VTU file
-	/// \returns The converted mesh or a nullptr if reading failed
+	/// \return The converted mesh or a nullptr if reading failed
 	static MeshLib::Mesh* readVTUFile(std::string const &file_name);
 
 	/// Writes the given mesh to file.
-	/// \returns True on success, false on error
+	/// Note: a MPI_Comm type argument is needed.
```

Comment:
Can you clarify this note. It is a bit misleading because there is no argument other than the filename to this method.